src/proto/grpc/testing/messages.proto

```diff
@@ -38,9 +38,6 @@ package grpc.testing;
 enum PayloadType {
   // Compressable text format.
   COMPRESSABLE = 0;
-
-  // Uncompressable binary format.
-  UNCOMPRESSABLE = 1;
```

Comment:
This will probably break some implementations. Do we really want to delete it or do we just not care if anyone does anything with it (since it isn't defined in the interop descriptions)?